CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.11...3.21)
+
+project(XoscarCollective)
+
+if(NOT ${PYTHON_EXECUTABLE})
```

Comment:
The `PYTHON_EXECUTABLE` may not be found, then the Python executable will be incorrect.

Reply:
Fix